AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Operations/Helpers/FetchAuthSessionOperationHelper.swift

```diff
@@ -0,0 +1,186 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import Amplify
+
+class FetchAuthSessionOperationHelper {
```

Comment:
Why is this a class? Do we have anything here to be accessed via reference? Else change to a struct.

Reply:
It keeps a reference of completion handler so that when the state machines finish, we could go back to the operation..